lld/wasm/InputChunks.cpp

```diff
@@ -385,6 +384,14 @@ bool InputChunk::generateRelocationCode(raw_ostream &os) const {
     if (!requiresRuntimeReloc)
       continue;
 
+    if (funcs.empty() || funcs.back().size() >= 7654300) {
```

Comment:
This hardcoded number probably warrants a commend and / or a name.

Reply:
This comes from `wasm_limits.h` [here](https://source.chromium.org/chromium/chromium/src/+/main:v8/src/wasm/wasm-limits.h;l=51)

Reply:
Perhaps lets like the to spec here instead of a specific impl.  How about: https://www.w3.org/TR/wasm-js-api-2/#limits?

Reply:
Good idea, it does list `7,654,321` for max function body

> The maximum size of a function body, including locals declarations, is 7,654,321 bytes.

Reply:
Updated